L1Trigger/L1TGEM/interface/ME0StubAlgoSubfunction.h

```diff
@@ -0,0 +1,78 @@
+#ifndef L1Trigger_L1TGEM_ME0StubAlgoSubfunction_H
+#define L1Trigger_L1TGEM_ME0StubAlgoSubfunction_H
+
+#include <cmath>
+#include <vector>
+#include <map>
+#include <cstdint>
+#include <algorithm>
+#include <bitset>
+#include <iostream>
+#include <numeric>
+#include "L1Trigger/L1TGEM/interface/ME0StubPrimitive.h"
+
+namespace l1t {
+  namespace me0 {
+    typedef std::bitset<192> UInt192;
+
+    struct Config {
+      bool skip_centroids;
```

Comment:
member variable names should also be formatted like `skipCentroids` (propagate everywhere)